stl/inc/algorithm

```diff
@@ -8901,7 +8907,7 @@ namespace ranges {
         }
 
         template <class _It, class _Pr, class _Pj>
-        static void _Buffered_merge_sort_common(const _It _First, const _It _Last, const iter_difference_t<_It> _Count,
+        static void _Buffered_merge_sort_common(const _It _First, const _It _Last, ptrdiff_t _Count,

```

Comment:
I don't see any introduced modifications, I think we could keep `_Count` `const` .